phpunit/style-engine/style-engine-test.php

```diff
@@ -676,4 +676,35 @@ public function test_should_dedupe_and_merge_css_rules() {
 
 		$this->assertSame( '.gandalf{color:white;height:190px;border-style:dotted;padding:10px;margin-bottom:100px;}.dumbledore,.rincewind{color:grey;height:90px;border-style:dotted;}', $compiled_stylesheet );
 	}
+
+	/**
+	 * Tests returning a generated stylesheet from a set of duotone rules.
```

Comment:
This is more of an integration test than a unit test for the style engine since it's testing both `get_id_selector_property_and_maybe_svg` and `gutenberg_style_engine_get_stylesheet_from_css_rules`. Is there a better place to put integration tests than mixed in with the unit tests for the style engine package?

Reply:
I updated the approach to just use strings.